pre_commit/languages/lua.py

```diff
@@ -0,0 +1,113 @@
+import contextlib
+import os
+import re
+from typing import Generator
+from typing import Sequence
+from typing import Tuple
+
+from pre_commit.envcontext import envcontext
+from pre_commit.envcontext import PatchesT
+from pre_commit.envcontext import Var
+from pre_commit.hook import Hook
+from pre_commit.languages import helpers
+from pre_commit.prefix import Prefix
+from pre_commit.util import clean_path_on_failure
+from pre_commit.util import cmd_output
+
+ENVIRONMENT_DIR = 'lua_env'
+get_default_version = helpers.basic_get_default_version
+healthy = helpers.basic_healthy
+
+
+def install_environment(
+    prefix: Prefix,
+    version: str,
+    additional_dependencies: Sequence[str],
+) -> None:
+    helpers.assert_version_default('lua', version)
+    # luarocks, Lua's package manager, is separate from the runtime.
+    helpers.assert_version_default('luarocks', version)
```

Comment:
this doesn't do anything -- you can remove this here -- the first parameter in this call here is just for the error message and doesn't do any additional checking